webrtc.html

```diff
@@ -2363,11 +2363,14 @@
                 which point the <code><a>RTCPeerConnection</a></code> can fully
                 adopt the pending local description, or rollback to the current
                 description if the remote side rejected the change.</p>
+                <p>As noted in <span data-jsep="ice-candidate-pool">[[!JSEP]]</span>,
+                calling this method triggers the ICE candidate gathering process
+                by the <a>ICE Agent</a>.</p>
                 <p>As noted in <span data-jsep="modifying-sdp">[[!JSEP]]</span>
                 the SDP returned from <code>createOffer</code> or
                 <code>createAnswer</code> MUST NOT be changed
-                before passing it to <code>setLocalDescription</code>. As
```

Comment:
I don't see why the "as a result" is being removed; it adds clarity, since it segues to how the "MUST" is enforced.